Tests/Tests/RouterSpec.swift

```diff
@@ -108,6 +114,38 @@ class RouterSpec: QuickSpec {
                             .to(containElementSatisfying({ $0 is ModalView }))
                     }
 
+                    it("will throw assertion if the campaign presenter cannot found") {
```

Comment:
It should be "...cannot be found". (Same for test names below)

Reply:
Fixed here [f60c555](https://github.com/rakutentech/ios-inappmessaging/pull/275/commits/f60c555517519f44c4bd47352520a9940b80058f).